PWGGA/GammaConv/AliAnalysisTaskGammaCaloMergedML.cxx

```diff
@@ -2572,6 +2598,14 @@ void AliAnalysisTaskGammaCaloMergedML::ProcessTrueClusterCandidatesAOD(AliAODCon
   Double_t mcProdVtxY   = primVtxMC->GetY();
   Double_t mcProdVtxZ   = primVtxMC->GetZ();
 
+  //Some Tree Variables
+  fMergedClusterTreeEnergy = cluster->E();
```

Comment:
also a bit of a general comment. I think when processing MC, you are interested in TruePt and recPt, not only truePt. This gives you an idea of detector response etc.